src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -135,7 +135,7 @@ private void ProcessCatalogItems(IndexWriter indexWriter, IEnumerable<JObject> c
 
             foreach (JObject catalogItem in catalogItems)
             {
-                _logger.LogInformation("Process CatalogItem {CatalogItem}", catalogItem["@id"]);
+                _logger.LogInformation($"Process CatalogItem {catalogItem["@id"]}", catalogItem["@id"]);
```

Comment:
reverted. Was for me to get local logging as this line loges Process CatalogItem [] to console logging.

---
In reply to: [75550188](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75550188) [](ancestors = 75550188)